src/dopri.ts

```diff
@@ -95,6 +97,7 @@ export class Dopri implements Integrator {
         this._stepper = new dopri5.Dopri5(rhs, n);
         this._control = dopriControl(control);
         this._output = output;
+        this._tcrit = [...this._control.tcrit];
```

Comment:
sorry couldn't find the answer easily but is this copied for a reason?

Reply:
this one no longer needed, there was a `pop()` on this before that is no longer there.